vendor/bat-native-ads/src/bat/ads/internal/eligible_ads/round_robin_ads.h

```diff
@@ -19,7 +21,9 @@ T FilterSeenAds(const T& ads, const std::map<std::string, bool>& seen_ads) {
 
   const auto iter = std::remove_if(
       unseen_ads.begin(), unseen_ads.end(), [&seen_ads](CreativeAdInfo& ad) {
-        return seen_ads.find(ad.creative_instance_id) != seen_ads.end();
+        SplitTestFrequencyCap frequency_cap;
+        return frequency_cap.ShouldExclude(ad) ||
```

Comment:
If an ad is excluded due to a split test it should not be considered for the round robin